### DISCUSSION TITLE
V, WinLibs and 'v doctor' ?

### DISCUSSION THREAD

**Discussion by ffred** [vlang/v#18831] [Questions and Answers] [answered]
Hi,
still exploring V and it's tools and just discovered 'v doctor' in a post on the forum.
I'm on Windows, with the Winlibs distribution installed and set in the PATH.

here is my 'v doctor' answer :
```
V full version: V 0.4.0 1728e4c
OS: windows, Microsoft Windows 8.1 v9600 64 bits
Processor: 4 cpus, 64bit, little endian, 

getwd: C:\Data\Dev\Vlang
vexe: C:\Programs\v\v.exe
vexe mtime: 2023-07-10 06:29:14

vroot: OK, value: C:\Programs\v
VMODULES: OK, value: C:\Users\FLT\.vmodules
VTMP: OK, value: C:\Users\FLT\AppData\Local\Temp\v_0

Git version: git version 2.37.3.windows.1
Git vroot status: weekly.2023.27-43-g1728e4c7
.git/config present: true

CC version: Error: exec failed (CreateProcess) with code 2: Le fichier spécifié est introuvable.

 cmd: cc --version
thirdparty/tcc status: thirdparty-windows-amd64 e90c2620
```
why does "v doctor" is looking for "cc" on Windows ? I'm really not an expert, but that's more a Linux command I think.. in my Winlibs DIR I have gcc, g++, clang..... but no cc.
is there a specific environment variable to set on Windows for 'v doctor' (or Vlang) to recognize the gcc install or is this cc info just an info and not really meaning much for V on Windows..?
thanks.

[EDIT: for info it's an old computer at work, but I got the same thing on my Win11 one at home..

**Comment by JalonSolov** [comment, marked as answer]
`cc` usually exists on every system _except_ Windows.  It is a link to the current default C compiler.  I would agree this needs to be fixed for Windows to avoid the error message, since Windows will likely never have this link.

**Comment by ffred** [comment]
ok, thanks. 
[Edit:] maybe we could define an environment variable on Windows to set the default compiler we want to use.

**Reply by spytheman** [reply]
You can set VFLAGS to `-cc msvc` or whatever other flags you want.

**Reply by JalonSolov** [reply]
That won't change what `v doctor` supports, and it will slow dev builds down, but it is an option.  :-)

**Reply by ffred** [reply]
didn't know - yet - about VFLAGS.. thanks, will look about it.
